spec.yaml

```diff
@@ -1174,6 +1173,22 @@ paths:
         }
         ```
 
+        ### Example request body for Azure
+
+        ```json
+        {
+          "provider": "azure",
+          "azure": {
+            "credential": {
+              "clientID": "c93bf55e-5bf7-4966-ad2b-e6f6e7721d50",
+              "secretID": "c2VjcmV0IHBhc3N3b3JkIGV4YW1wbGUK",
```

Comment:
By default Azure uses passwords (secret key) in UUID format. Or i'm missing smth and this is base64 encoded?

Reply:
default is UUID, by you can also have it base64 encoded. Not even sure how to get one or the other

Reply:
changed to UUID